airdrop/getter.gno

```diff
@@ -0,0 +1,125 @@
+package airdrop
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/json"
+	"gno.land/p/demo/ufmt"
+	"gno.land/p/demo/uint256"
+)
+
+// GetAirdropConfig returns the current configuration of the airdrop as a JSON string.
+//
+// Example:
+//
+//	{
+//	  "refundable_timestamp": "1234567890",
+//	  "refund_to": "g1234567890abcdefghijklmnop"
+//	}
+func (a *Airdrop) GetAirdropConfig() string {
+	node := json.ObjectNode("", nil)
+
+	timestamp := strconv.FormatUint(a.config.RefundableTimestamp, 10)
+	err := node.AppendObject("refundable_timestamp", json.StringNode("refundable_timestamp", timestamp))
+	if err != nil {
+		panic(err)
+	}
+
+	err = node.AppendObject("refund_to", json.StringNode("refund_to", a.config.RefundTo.String()))
+	if err != nil {
+		panic(err)
+	}
+
+	return marshal(node)
+}
+
+// GetTotalClaims returns the total number of claims made so far as a JSON string.
+//
+// Example:
+//
+//	{
+//	  "total_claims": "16"
+//	}
+func (a *Airdrop) GetTotalClaims() string {
+	totalClaims := 0
+	for _, bitmap := range a.claimedBitmap {
+		totalClaims += popCount(bitmap)
+	}
+
+	node := json.ObjectNode("", nil)
+
+	claims := strconv.Itoa(totalClaims)
+	err := node.AppendObject("total_claims", json.StringNode("total_claims", claims))
+	if err != nil {
+		panic(err)
+	}
+
+	return marshal(node)
+}
+
+// GetRemainingBalance returns the remaining balance of tokens available for airdrop.
+//
+// Example:
+//
+//	{
+//	  "remaining_balance": "1000000"
+//	}
+func (a *Airdrop) GetRemainingBalance() string {
```

Comment:
make it primitive

Reply:
https://github.com/gnoswap-labs/governance/pull/19/commits/a24003903330ffb7ba3e1f7b0180a6e6e8da951e

Reply:
same as above 
https://github.com/gnoswap-labs/governance/pull/20/commits/b1335b03623b74940851e9db25146e9218be526d